infrastructure/validators/operational-tasks/chill-your-node.md

```diff
@@ -0,0 +1,79 @@
+continues to be chilled for the rest of the era, it would NOT be selected for election in the next
+era. If a bond was chilled for the entire session 5, it would not be considered in the next
+election. If the bond was chilled in session 6, its participation in the next era's election would
+depend on its state in session 5.
+
+## Chilling as a Nominator
+
+When you chill after being a nominator, your nominations will be reset. This means that when you
+decide to start nominating again you will need to select validators to nominate once again. These
+can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations
+will not persist across chills.
+
+Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will
+not need to go through the whole process of bonding again, rather, you will issue a new nominate
+call that specifies the new validators to nominate.
+
+## Chilling as a Validator
+
+When you voluntarily chill after being a validator, your nominators will remain. As long as your
+nominators make no action, you will still have the nominations when you choose to become an active
+validator once again. You bond however would not be listed as a nominable validator thus any
+nominators issuing new or revisions to existing nominations would not be able to select your bond.
+
+When you become an active validator, you will also need to reset your validator preferences
+(commission, etc.). These can be configured as the same values set previously or something
+different.
+
+## Chill Other
+
+An unbounded and unlimited number of nominators and validators in Polkadot's NPoS is not possible
+due to constraints in the runtime. As a result, multiple checks are incorporated to keep the size of
+staking system manageable, like mandating minimum active bond requirements for both nominators and
+validators. When these requirements are modified through on-chain governance, they can be enforced
+only on the accounts that newly call `nominate` or `validate` after the update. The changes to the
+bonding parameters would not automatically chill the active accounts on-chain which do not meet the
+requirements.
+
+!!!note "Chill Threshold"
+    `ChillThreshold` defines how close to the max nominators or validators we must reach before users
+    can start chilling one another.
+
+For instance, let us consider a scenario where the minimum staking requirement for nominators is
+changed from 80 DOTs to 120 DOTs. An account that was actively nominating with 80 DOTs before this
+update would still keep receiving staking rewards. To handle this corner case, the `chillOther`
+extrinsic was incorporated which also helps to keep things backwards compatible and safe. The
+`chillOther` extrinsic is permissionless and any third party user can target it on an account where
+the minimum active bond is not satisfied, and chill that account. The list of addresses of all the
```

Comment:
```suggestion
the minimum active bond isn't satisfied, and chill that account. The list of addresses of all the
```
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.